src/components/MultipieceModal/MultipieceModal.js

```diff
@@ -12,22 +12,41 @@ import {
 } from '@folio/stripes/components';
 
 const MultipieceModal = (props) => {
-  const { item, open, onClose, onConfirm } = props;
-  const { title, barcode, materialType } = item;
-
+  const {
+    item,
+    open,
+    onClose,
+    onConfirm,
+  } = props;
+  const {
+    title,
+    barcode,
+    materialType,
+  } = item;
```

Comment:
Could it be better if we combine both destructurings and put them inside props?

Reply:
unfortunately, no.
There is a case when `item` is passed as an argument (see the next line of the code)